software/affine.yml

```
@@ -0,0 +1,12 @@
+name: "AFFiNE"
```

```suggestion
name: "AFFiNE Community Edition"
```
As per previous discussions https://github.com/awesome-selfhosted/awesome-selfhosted-data/issues/1200